block.go

```diff
@@ -769,7 +764,9 @@ func (p *parser) table(data []byte) int {
 		}
 
 		// include the newline in data sent to tableRow
-		i++
+		if i < len(data)-1 && data[i] == '\n' {
```

Comment:
This must've been a copypasta, the tests still pass with `-1` removed. Thanks for catching :-)